src/main/java/io/jenkins/plugins/jfrog/CliEnvConfigurator.java

```diff
@@ -79,4 +79,11 @@ private static void excludeProxyEnvFromPublishing(EnvVars env) {
         String jfrogCliEnvExclude = env.getOrDefault(JFROG_CLI_ENV_EXCLUDE, JFROG_CLI_DEFAULT_EXCLUSIONS);
         env.put(JFROG_CLI_ENV_EXCLUDE, String.join(";", jfrogCliEnvExclude, HTTP_PROXY_ENV, HTTPS_PROXY_ENV));
     }
+
+    static String noProxyExtractor(String noProxyList) {
```

Comment:
Let's add a short JavaDoc comment above this method (see the documentation in excludeProxyEnvFromPublishing for reference)